site/layouts/shortcodes/scss-docs.html

```diff
@@ -42,7 +42,7 @@
   <div class="bd-example-snippet bd-code-snippet bd-scss-docs">
     <!--Boosted mod: added `border-light border-1 border-top-0`-->
     <div class="d-flex align-items-center highlight-toolbar ps-3 pe-2 py-1 border-bottom border-light border-1 border-top-0">
-      <a class="font-monospace text-body-secondary text-decoration-none small" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/{{ $file | replaceRE `\\` "/" }}">
+      <a class="font-monospace link-secondary link-underline-secondary link-underline-opacity-0 link-underline-opacity-100-hover small" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/{{ $file | replaceRE `\\` "/" }}">
```

Comment:
Seems to be more accessible
```suggestion
      <a class="font-monospace link-body-emphasis link-underline-secondary link-underline-opacity-0 link-underline-opacity-100-hover small" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/{{ $file | replaceRE `\\` "/" }}">
```

Reply:
I merged it since there are no options with utilities but we should have something that allows keeping the same color. Like a `.link-body` maybe IDK.